cli/cli-client/src/main/java/org/infinispan/cli/commands/CliCommand.java

```diff
@@ -28,7 +28,7 @@
    public CommandResult execute(ContextAwareCommandInvocation invocation) throws CommandException {
       if (help) {
          invocation.println(invocation.getHelpInfo());
-         return CommandResult.SUCCESS;
+         return CommandResult.FAILURE;
```

Comment:
why this change?

Reply:
My mistake. I thought Unix commands returned failures when invoked with the help switch. Instead they all return 0, so I'll revert this.